content/pt-br/docs/setup/production-environment/tools/kops.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Instalando Kubernetes com kOps
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+Este início rápido mostra como instalar facilmente um cluster Kubernetes na AWS usando uma ferramenta chamada [`kOps`](https://github.com/kubernetes/kops). 
+
+`kOps` é um sistema de provisionamento automatizado:
+
+* Instalação totalmente automatizada
+* Usa DNS para identificar clusters
+* Auto-recuperação: tudo é executado em grupos de Auto-Scaling
+* Suporte de vários sistemas operacionais (Amazon Linux, Debian, Flatcar, RHEL, Rocky e Ubuntu) - veja em [imagens](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md)
+* Suporte a alta disponibilidade - consulte a [documentação sobre alta disponibilidade](https://github.com/kubernetes/kops/blob/master/docs/operations/high_availability.md)
+* Pode provisionar diretamente ou gerar manifestos do terraform - veja o [veja a documentação sobre como fazer isso com Terraform](https://github.com/kubernetes/kops/blob/master/docs/terraform.md)
+
+## {{% heading "prerequisites" %}}
+
+* Você deve ter o [kubectl](/docs/tasks/tools/) instalado.
+
+* Você deve [instalar](https://github.com/kubernetes/kops#installing) `kops` em uma arquitetura de dispositivo de 64 bits (AMD64 e Intel 64).
+
+* Você deve ter uma [conta da AWS](https://docs.aws.amazon.com/polly/latest/dg/setting-up.html), gerar [chaves do IAM](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) e [configurá-las](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#cli-quick-configuration). O usuário do IAM precisará de [permissões adequadas](https://github.com/kubernetes/kops/blob/master/docs/getting_started/aws.md#setup-iam-user).
+
+<!-- steps -->
+
+## Como criar um cluster
+
+### (1/5) Instalar kops
+
+#### Instalação
+
+Faça o download do kops na [página de downloads](https://github.com/kubernetes/kops/releases) (também é conveniente compilar a partir da fonte):
+
+{{< tabs name="instalação_kops" >}}
```

Comment:
The tabs are broken to me. Probably because the name was translated. @stormqueen1990 can the tab name be translated?

Reply:
I had seen it translated elsewhere but restored to the English version.

Reply:
Now, it's working. 💯